packages/components/components/elvis-header/src/react/sideNav/publicStyles.tsx

```diff
@@ -50,7 +50,8 @@ export const publicStyles = (isGtMobile: boolean) => {
       flex: none;
       border: 1px solid transparent;
 
-      i[class*='color'] {
+      i[class*='color'],
+      e-icon[name*='color' i] {
```

Comment:
Is the `[name*='color' i` selector case-insensitive?

Reply:
Yep, that's that the `i` does!